.gitignore

```diff
@@ -25,3 +25,8 @@ __pycache__
 
 # Emacs backup
 *~
+
+# VSCode
```

Comment:
Could we pull this out into it's own ticket?   DOn't want to confuse things by having the VSCode default mixed in...

Reply:
Then, I think it's good for merging.